scripts/2.mutation-classifier.py

```diff
@@ -234,21 +289,26 @@ def select_feature_set_columns(X, feature_set):
     cv_results_df = cv_results_df.append(df)
 
 
-# In[17]:
+# In[18]:
+
 
 # Cross-validated performance heatmap
-cv_score_mat = pd.pivot_table(cv_results_df,
-                              values='mean_test_score', 
-                              index='feature_set',
-                              columns='classify__alpha')
-ax = sns.heatmap(cv_score_mat, annot=True, fmt='.1%')
-ax.set_xlabel('Regularization strength multiplier (alpha)')
-ax.set_ylabel('Feature Set');
+cv_results_df['classify__alpha'] = cv_results_df['classify__alpha'].astype(str)
+
+(ggplot(cv_results_df, aes(x='classify__alpha',
+                           y='mean_test_score',
+                           fill='feature_set'))
+ + geom_bar(stat='identity', position='dodge')
```

Comment:
I switched to a scatterplot and the log scale -- it initially looked funky to me with the default theme. I used a hack to get 'expand limits'.